### PR TITLE
task 6 

### PR DESCRIPTION
**Pull request by Julia-ivv**
- **done** - File **serverless.yml** contains configuration for **catalogBatchProcess** function
- **done** - File **serverless.yml** contains policies to allow lambda **catalogBatchProcess** function to interact with SNS and SQS
- **done** - File **serverless.yml** contains configuration for SQS **catalogItemsQueue**
- **done** - File **serverless.yml** contains configuration for SNS Topic **createProductTopic** and email subscription

- **done** - set a Filter Policy for SNS **createProductTopic** in **serverless.yml**

### CODE REVIEW

**Review by vvkotov — APPROVED** [review]

File: import-service/handler.js [hunk 1]
```diff
@@ -1,5 +1,7 @@
 const S3 = require('aws-sdk/clients/s3');
+const SQS = require('aws-sdk/clients/sqs');
 const csv = require('csv-parser');
+// const csv = require('csvtojson');
```
Comment (balabukha) [inline import-service/handler.js:4]: please delete unused comments

File: import-service/package.json [hunk 2]
```diff
@@ -11,6 +11,7 @@
   "dependencies": {
     "aws-sdk": "^2.792.0",
     "csv-parser": "^2.3.3",
+    "csvtojson": "^2.0.10",
```
Comment (balabukha) [inline import-service/package.json:14]: it's unused, can be deleted

File: product-service/handlers/catalogBatchProcess.js [hunk 3]
```diff
@@ -0,0 +1,76 @@
+import AWS from 'aws-sdk';
```
Comment (balabukha) [inline product-service/handlers/catalogBatchProcess.js:1]: better to write 
`import {SNS} from 'aws-sdk'`;

File: product-service/handlers/catalogBatchProcess.js [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+import { Client } from 'pg';
+import { dbOptions } from '../constants/dbOptions';
+import { headers } from '../constants/headers';
+import { isValid } from '../utils/productValidation';
+
+export const catalogBatchProcess = async event => {
+    const sns = new AWS.SNS({ region: 'eu-west-1' });
+    const client = new Client(dbOptions);
+    await client.connect();
```
Comment (balabukha) [inline product-service/handlers/catalogBatchProcess.js:10]: the logic for connecting to DB can be moved to common

**Review by balabukha — APPROVED** [review]

**Pull request merged**